README.md

```diff
@@ -31,11 +31,7 @@ type IntOrA[A] = Either[Int, A]
 
 Many people have wished for a better way to do this.
 
-The goal of this plugin is to add a syntax for type lambdas. We do
-this by rewriting syntactically valid programs into new programs,
-letting us seem to add new keywords to the language. This is achieved
-through a compiler plugin performing an (un-typed) tree
-transformation.
+The goal is to add a syntax for type lambdas. Rewriting syntactically valid programs into new programs which seems to add new keywords to the language. This is achieved through a compiler plugin performing an (un-typed) tree transformation.
```

Comment:
I appreciate to effort to improve the documentation. Most people, after they finally get what the author tried to say, don't come back with feedback how things could have been phrased better. 👍 

The structure of the following sentence doesn't seem right, though:

> Rewriting syntactically valid programs into new programs which seems to add new keywords to the language.